src/server.js

```diff
@@ -258,6 +262,10 @@ const respondValueSearch = (
     });
   }
 
+  // Save response to database
+  const record = createColorRecord({ paletteTitle, colors: colorsResponse, list: listKey });
```

Comment:
can we also add the IP of the request? Would be soo cool to be able to do popular colors by country or so!

Reply:
IP addresses are be considered personal data so can't store them as is. But we could store a hash.